yarn.lock

```diff
@@ -812,18 +812,16 @@ acorn@^8.9.0:
 
 agent-base@6, agent-base@^6.0.2:
   version "6.0.2"
-  resolved "https://registry.npmjs.org/agent-base/-/agent-base-6.0.2.tgz"
+  resolved "https://registry.yarnpkg.com/agent-base/-/agent-base-6.0.2.tgz#49fff58577cfee3f37176feab4c22e00f86d7f77"
```

Comment:
The yarn lock update needed or just boyscouting?

Neither is an issue, just curious for context

Reply:
This, happened automatically when I ran yarn install.